src/Microsoft.ML.Transforms/Text/LdaTransform.cs

```diff
@@ -141,6 +141,9 @@ internal sealed class Column : OneToOneColumn
             [Argument(ArgumentType.AtMostOnce, HelpText = "Reset the random number generator for each document", ShortName = "reset")]
             public bool? ResetRandomGenerator;
 
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Whether to output the topic-word summary in text format", ShortName = "summary")]
```

Comment:
nit: Suggestion, change the HelpText of this argument and the one on `internal Options.OutputTopicWordSummary` to be "Whether to output the topic-word summary in text format when saving the model to disk"`, else it isn't understood that users will actually need to save the model to disk in order to get that output. (see https://github.com/microsoft/NimbusML/issues/405 where the user actually expected to see the output on the console).

Reply:
sounds good, thanks

---
In reply to: [445990136](https://github.com/dotnet/machinelearning/pull/5260#discussion_r445990136) [](ancestors = 445990136)